addon/ng2/blueprints/ng2/files/__path__/vendor.ts

```diff
@@ -0,0 +1,17 @@
+// Typescript emit helpers polyfill
+import 'ts-helpers';
+
+import '@angular/core';
```

Comment:
By putting this imports at vendor we are able to extract a cached vendor file with all vendor dependencies. That way you can update your app bundle without having to load the vendors again.